packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -845,7 +868,7 @@ export default defineComponent({
 })
 </script>
 <style lang="scss">
-.oc-table.condensed > tbody > tr {
+.condensed > .oc-table > tbody > tr {
```

Comment:
@elizavetaRa ping

Reply:
With the collapsible table, condensed is applied to a div, that in turn contains the dropdown selector and the oc-table, instead of bring applied to oc-table.

Reply:
With the collapsible table, condensed is applied to a div, that in turn contains the dropdown selector and the oc-table, instead of bring applied to oc-table.